### DISCUSSION TITLE
Cutoff Date no longer revokes permissions...

### DISCUSSION THREAD

**Discussion by mcknijak** [community/community#103264] [GitHub Education]
### Hi everyone,

Hello,

We're trying to use the new beta features in my course, but can anyone point to actual bug logging and updates on what they're working on fixing?

My students had their first deadline last night and this morning I logged in to discover they all still had access to the assignment repos, even though the deadline had passed and the cutoff date box was selected.

The classroom bot also doesn't automatically add its normal commits - like loading the autograder config - when they accept the assignment. 

It feels like these things should've been caught and tested before moving to the beta stage. At the very least, a better way of issue tracking the update would be nice.

Anyone have information on when things like this will be fixed?

**Comment by aachandler-wcu** [comment]
Wanted to check in on this and see if anyone had gotten in contact with you about this. It seems we are running into the "cutoff date'
 not revoking privileges issue in our classrooms now, and I also can't find any information about it. I submitted a ticket yesterday, but currently every student is able to manipulate their repo's after the cutoff date without any issues and we don't have any workarounds or fixes. They are at least marked late, but it looks like every repo in the assignment is also marked as late and we are not totally sure if that is related.

**Reply by michaelnetter** [reply]
+1
